csharp/src/Ice/Internal/MetricsAdminI.cs

```diff
@@ -182,9 +182,9 @@ internal IceMX.MetricsFailures getFailures()
         {
             if (_failures == null)
             {
-                return null;
```

Comment:
This was bogus. We no longer accept null for struct parameters.

Reply:
Why is the generated type a record class and not a record struct?

```
public sealed partial record class MetricsFailures
```

Is this bogus, or did we change the default mapping for structs to record class?

Reply:
This is not bogus. In C#, the default mapping for most Slice structs, including `struct MetricsFailures`, is a C# class. For Ice 3.8, it's a "record class" instead of a plain class.